server/handlers/result_handlers.py

```diff
@@ -49,6 +49,31 @@ def __call__(self, result_batch: list[dict]) -> None:
         """
         pass
 
+    def prepare_result(self, result: Dict) -> "LSEBatchItem":
```

Comment:
I think it might be more obvious if we make this part of `LSEBatchItem`, like a `LSEBatchItem.from_result` class method. Wdyt?

Reply:
done